card/lib/card/auth/current.rb

```diff
@@ -49,6 +49,21 @@ def current= mark
           end
       end
 
+      def serialize
+        { as_id: as_id, current_id: current_id }
+      end
+
+      def deserialize data
```

Comment:
do we have any kind of naming pattern to suggest a method expects a block?   eg `deserializing`?